src/test/java/com/gooddata/JsonMatchers.java

```diff
@@ -37,6 +41,16 @@ private JsonMatchers() {
         return new JsonFileMatcher<>(expectedResourceName);
     }
 
+    /**
+     * Creates a matcher that matches String against JSON in given resource file.
+     * @param expectedResourceName name of resource (relative patch in resource folder)
+     * @return  matcher that matches String against JSON in given resource file
+     */
+    @Factory
+    public static Matcher<String> isJsonString(final String expectedResourceName) {
```

Comment:
The "String" in name is a bit misleading since it compares the JSON deeply, not only "String comparision" ;)